File: scripts/audit/fixtures/PAN003-kind-segment-parity/good/scripts/lib/kinds.py
"""Fixture (PAN003-kind-segment-parity / good): Python kind->segment SoT that
AGREES with the sibling src/lib/kinds.ts. The audit execs this module, so it
must import."""

SEGMENT_OF: dict[str, str] = {"book": "books", "poem": "poetry", "project": "projects"}

File: audit/fixtures/PAN003-kind-segment-parity/good/src/lib/kinds.ts
// Fixture (PAN003-kind-segment-parity / good): TS kind→segment SoT that AGREES
// with the sibling pancratius/kinds.py — the cross-language audit must stay
// silent.
type WorkKind = "book" | "poem" | "project";
type WorkSegment = "books" | "poetry" | "projects";
export const SEGMENT_OF: Record<WorkKind, WorkSegment> = {
  book: "books",
  poem: "poetry",
  project: "projects",
};
